modules/mo_pytorch/mo_extensions/front/pytorch/elementwise_ext.py

```diff
@@ -23,6 +23,8 @@
 from mo.ops.power import AttributedPower
 from extensions.ops.activation_ops import *
 from mo.ops.const import Const
+from mo.ops.clamp import Clamp
```

Comment:
Unused import

Reply:
Done, thanks! We need to enable a static analyzer in future.